src/pypi_changes/_cli.py

```diff
@@ -43,6 +44,18 @@ def _define_cli_arguments() -> ArgumentParser:
     cache_help = "seconds how long requests should be cached (pass 0 to bypass the cache, -1 to cache forever)"
     parser.add_argument("--cache-duration", "-d", default=3600, type=int, help=cache_help, metavar="SEC")
 
+    parser.add_argument(
+        "--sort",
+        "-s",
+        help="Sorting method",
+        choices=["a", "alphabetic", "u", "updated"],
+        default="updated",
+        dest="sort",
+        const="updated",  # noqa: SC200
```

Comment:
What gives here?

Reply:
flake8 is failing, and I'm not sure why in this case, because `const` is an argument here
```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

src/pypi_changes/_cli.py:54:9: SC200 Possibly misspelt word: 'const'
```

Reply:
You need to add it to https://github.com/gaborbernat/pypi_changes/blob/main/whitelist.txt